build.gradle

```diff
@@ -30,21 +15,28 @@ ext {
     fregeBaseVersion = "3.23.288"
     fregeClassifier = "-gaa3af0c"
     fregeVersion = "$fregeBaseVersion$fregeClassifier"
+
+    // work around https://issues.gradle.org/browse/GRADLE-3281
+    def home      = new File(System.getProperty("user.home"))
+    def propfile  = new File(home, ".gradle/gradle.properties")
+    if (propfile.exists()) {
+        props = new Properties()
+        propfile.withReader {
+            props.load(it)
```

Comment:
If the .gradle/gradle.properties file exists, but the key "gradle.publish.key" does not there is a NullPointerException.  This is what I get.